tests/CMakeLists.txt

```diff
@@ -1,10 +1,11 @@
-# This is the cmake build file for the tests directory of the
-# NCEPLIBS-g2c_UTILS project.
-#
-# Ed Hartnett, 7/11/21
```

Comment:
Please do not remove authorship information. If you make significant contribution, add your name.

Reply:
Git contains all authorship information. We do not need list every single file change in the comments.

Reply:
Please do not remove authorship information. Thanks.